pkg/puppet/puppet.go

```diff
@@ -255,6 +271,11 @@ func contentClusterConfig(cluster interfaces.Cluster) ([]string, error) {
 		hieraData.variables = append(hieraData.variables, fmt.Sprintf(`tarmak::fluent_bit_configs: %s`, string(jsonLoggingSink)))
 	}
 
+	if packerBuild {
+		hieraData.variables = append(hieraData.variables, `vault_client::run_exec: false`)
```

Comment:
We should only need one variable here that we pass to the tarmak class. vault_client inherits tarmak.

Reply:
We already set a bunch of other `vault_client` vars with hiera. https://github.com/jetstack/tarmak/blob/master/puppet/hieradata/common.yaml#L20-L25

AFAIK you need to choose between setting everything with hiera or everything inside a Puppet class like:
```
  class { 'vault_client':
     run_exec   => false,
     init_token => 'sometoken',
     ...    
  }
```

If we would move `vault_client::run_exec` to the Tarmak class, that would mean that we also need to make extra changes for the variables we have now in hieradata for `vault_client`.